### PR TITLE
Fix/OP Accept missing data analysis in customers mip json

### PR DESCRIPTION
**Pull request by patrikgrenfeldt**
This PR fixes the problem handling mip-json without specified data_analysis 

### How to prepare for test
- [x] ssh to clinical-db (depending on type of change)
- [x] install on stage:
`bash /home/proj/production/servers/resources/clinical-db.scilifelab.se/update-clinical-api-stage.sh [THIS-BRANCH-NAME]`

### How to test
- [x] login to OP-stage
- [x] submit an actual customer mip-json

### Expected test outcome
- [x] check that the order is accepted
- [x] Take a screenshot and attach or copy/paste the output.

## Review
- [x] code approved by MR
- [x] tests executed by PG
- [x] "Merge and deploy" approved by PG
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a
- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [x] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan
- [ ] Deploy this branch
- [ ] Inform to production, lab

### CODE REVIEW

File: cg/apps/lims/limsjson.py [hunk 1]
```diff
@@ -19,7 +19,7 @@
 def get_project_type(samples: [dict]) -> str:
     """Determine the project type."""
 
-    data_analyses = set(sample["data_analysis"].lower() for sample in samples)
+    data_analyses = set(sample.get("data_analysis", "mip-dna").lower() for sample in samples)
```
Comment (Mropat) [inline cg/apps/lims/limsjson.py:22]: Pipeline value here?

**Review by Mropat — APPROVED** [review]
Approved! But since we implemented enum for pipelines, we should be using it to get the mip-dna value correctly!

**Comment by patrikgrenfeldt** [comment]
Test result:
image

👍

File: cg/apps/lims/limsjson.py (hunk 1 above)
Comment (patrikgrenfeldt) [reply cg/apps/lims/limsjson.py:22]: I'm working on adding Balsamic, I'll add it in that PR

**Comment by patrikgrenfeldt** [comment]
Deployed on clinical-db:
image

**Pull request merged**